examples/plugin.cpp

```diff
@@ -203,15 +191,14 @@ class kimera_vio : public plugin {
     const std::shared_ptr<switchboard> sb;
     switchboard::writer<pose_type> _m_pose;
     switchboard::writer<imu_integrator_input> _m_imu_integrator_input;
-    time_type _m_begin;
 
     VIO::FrameId kimera_current_frame_id;
     VIO::VioParams kimera_pipeline_params;
     VIO::Pipeline kimera_pipeline;
     
     //const imu_cam_type* imu_cam_buffer;
     switchboard::ptr<const imu_cam_type> imu_cam_buffer;
-    double previous_timestamp = 0.0;
+    time_point previous_timestamp = time_point{duration{-1}};
```

Comment:
Initialize in constructor?

Reply:
For stuff whose initialization is not dependent on constructor params, clang-tidy recommends initializing there.